<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Receiver log</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>已接收帧信息</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>20</width>
     <height>331</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="table2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;JetBrains Mono&quot;;</string>
   </property>
   <column>
    <property name="text">
     <string>序列号</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数据</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>发送</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="table3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>121</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;JetBrains Mono&quot;;</string>
   </property>
   <column>
    <property name="text">
     <string>数据</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="button_time">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>0</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据超时模拟</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
